internal/gps/lock.go

```diff
@@ -195,6 +196,15 @@ func (lp LockedProject) Packages() []string {
 	return lp.pkgs
 }
 
+func (lp LockedProject) String() string {
+	prj := lp.Ident()
+	rev, _, _ := VersionComponentStrings(lp.Version())
```

Comment:
Since `Version` defines `Stringer` instead of `VersionComponentStrings`, I directly used it, should i edit `versionPairs` Stringer definition to include both version and rev.